projects/github.com/restic/restic/package.yml

```diff
@@ -0,0 +1,22 @@
+distributable:
+  url: https://github.com/restic/restic/releases/download/v{{ version }}/restic-{{version}}.tar.gz
+  strip-components: 1
+
+display-name: restic
+
+versions:
+  github: restic/restic
+
+provides:
+  - bin/restic
+
+build:
+  script: |
+    go run build.go
+    mkdir -p "{{ prefix }}"/bin
+    mv restic "{{ prefix }}"/bin
+  dependencies:
+    go.dev: ^1.20
+
+test:
+  restic version
```

Comment:
Why not run `go test ./...`?

Reply:
the source isn't available in the `test` node. if you wanted to run source tests, you'd do that in the `build` node. the `test` node is for ensuring the resulting binary(ies) work properly.

Reply:
note that I really should make the sources and build dirs available since this is a common thing we want to do